.env.example

```diff
@@ -3,10 +3,10 @@ APP_DEBUG=true
 APP_URL=http://localhost
 APP_KEY=SomeRandomString
 
-DB_DRIVER=sqlite
+DB_DRIVER=mysql
 DB_HOST=localhost
 DB_DATABASE=cachet
-DB_USERNAME=user
+DB_USERNAME=homestead
```

Comment:
The reason for changing the defaults is so that it is setup for development by default. In any production environment, these would get overwritten anyway.

Not sure if that makes sense.

We can change the defaults back, and then add more documentation explaining what to change - that would be the other option.